processing/src/main/java/org/apache/druid/segment/data/CompressedVSizeColumnarIntsSupplier.java

```diff
@@ -308,7 +308,7 @@ protected int _get(ByteBuffer buffer, boolean bigEngian, int bufferIndex)
     }
   }
 
-  private class CompressedVSizeColumnarInts implements ColumnarInts
+  public class CompressedVSizeColumnarInts implements ColumnarInts
```

Comment:
Since CompressedVSizeColumnarInts is the parent class of the other classes in the file. It would be enough for most use cases to have this as public since it could expose methods which the child classes can override.